db/migrate/20170603022915_adds_issued_at_to_donation.rb

```diff
@@ -1,12 +1,16 @@
 class AddsIssuedAtToDonation < ActiveRecord::Migration[5.0]
+  class MigrationDonation < ActiveRecord::Base
+    self.table_name = :donations
+  end
+
   # doin this old-school because we need to initialize it programmatically
   def up
```

Comment:
I wonder if we should have some puts or logs so when a user runs `rake db:migrate` they can see that something happened? I think folks are used to seeing some sort of output from a migration.